lib/Wikia/src/Persistence/User/Attributes/AttributePersistenceSwagger.php

```diff
@@ -120,16 +120,16 @@ private function getApi( $userId ) {
 	private function handleApiException( ApiException $e ) {
 		switch ( $e->getCode() ) {
 			case UnauthorizedException::CODE:
-				throw new UnauthorizedException();
+				throw new UnauthorizedException( $e->getResponseBody() );
```

Comment:
`$e->getResponseBody()` has the data from the response we're actually interested in. It includes the HTTP response code, as well as the message returned by the attribute service.